pkg/token/service.go

```diff
@@ -59,7 +59,7 @@ type RefreshTokenData struct {
 }
 
 type tokenService struct {
-	tokenRepository               Repository
+	tokenRepository               repository
```

Comment:
since we are already in the token package I think we don't need the `token` prefix on the `repository` field